### DISCUSSION TITLE
Additional Functionality for report

### DISCUSSION THREAD

**Discussion by JayMMAI24** [fbdesignpro/sweetviz#169] [Ideas]
I was wondering if there is a way to include the Pandas column type in the report? Maybe something that looks like this:

![image](https://github.com/fbdesignpro/sweetviz/assets/156379723/130f3594-e5d3-4230-9ec9-3f0de0891bd6)

The reason for the suggestion is that even after using the report, I still have to use .info() to get a list of column types, so perhaps integrating it as part of the report can eliminate that need?

**Comment by fbdesignpro** [comment]
Hi @JayMMAI24, that's an interesting suggestion, and shouldn't be too hard to implement. I don't have a lot of time these days but I'll keep this on the list.
